docs/plugins/embyupdate.rst

```diff
@@ -30,4 +31,5 @@ The available options under the ``emby:`` section are:
 - **port**: The Emby server port.
   Default: 8096
 - **username**: A username of a Emby user that is allowed to refresh the library.
-- **password**: That user's password.
+- **apikey**: An Emby API key
+- **password**: If no API key is present then enter the user aboves password.
```

Comment:
this needs rewording. Perhaps something like.  Emby password (only needed if no API key is provided)